src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,12 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import core.mate.academy.model.Machine;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Machine> {
```

Comment:
why do we parameterize this class with `Machine`? If the name of the class tell us that we will return only `Bulldozers`?

Reply:
done